e2e/benchmarks/model_config.js

```diff
@@ -216,7 +219,11 @@ const benchmarks = {
         };
       }
       const model = await posenet.load(config);
-      model.image = await loadImage('tennis_standing.jpg');
+      if (inputType === 'tensor') {
+        model.image = tf.zeros([inputResolution, inputResolution, 3]);
```

Comment:
nit: Prefer renaming to `model.input = ` rather than `model.image`. Same with other places.

Reply:
Done, PTAL.